.gitpod.yml

```
@@ -0,0 +1,7 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+    - init: pnpm install && pnpm run build
+      command: pnpm run start && chmod +x .husky/pre-commit && chmod +x .husky/pre-push
```

```suggestion
    - init: pnpm i && pnpm build
      command: pnpm start
```
1. 命令简写了一下
2. Shell 脚本执行一次 `chmod` 再提交，后面就无需再执行，类 UNIX 系统的权限会存在 Git 里

---

那我以后每次新开一个仓库的workspace都需要手动执行一次 `chmod` 吗？直接放在初始化里面会不会更好一些？

---

> 2. Shell 脚本执行一次 `chmod` 再提交，后面就无需再执行，类 UNIX 系统的权限会存在 Git 里

看清楚“权限存在 Git 里”、“无需再执行”